src/metabase/driver/duckdb.clj

```diff
@@ -225,9 +225,26 @@
     (.getResultSet stmt)
     (empty-rs)))
 
+(defn- is_motherduck_single_mode 
+  [database_file]
+  (and (seq (re-find #"^md:" database_file)) (> (count database_file) 3)))
```

Comment:
so it's possible in Metabase to connect to just "md:"? The experience is probably not great...

Reply:
not specifying a database might actually fail because of the motherduck_attach_mode config.

Reply:
oh yeah, it should. Should we be checking `attach_mode` instead of checking that the connection string specifies a single database?

Reply:
it's always enabled for motherduck connections. maybe this check should just be called is_motherduck

Reply:
makes sense!